mock-core-configs/etc/mock/alma-8-x86_64.cfg

```diff
@@ -0,0 +1,24 @@
+include('/etc/mock/templates/alma-8.tpl')
+include('/etc/mock/templates/epel-8.tpl.rpmnew')
+
+config_opts['root'] = 'alma-8-x86_64'
+config_opts['target_arch'] = 'x86_64'
+config_opts['legal_host_arches'] = ('x86_64',)
+config_opts['hostname'] = 'mock'
+config_opts['environment']['HOSTNAME'] = 'mock'
+
+config_opts['files']['etc/dnf/modules.d/virt.module'] = """[virt]
+name=virt
+profiles=
+state=disabled
+stream=
+"""
+
+config_opts['releasever'] = '8'
+config_opts['package_manager'] = 'dnf'
+
+config_opts['use_bootstrap'] = False
+config_opts['use_bootstrap_image'] = False
+
+config_opts['chroot_setup_cmd'] += ' rpm-build nano dnf dnf-plugins-core glibc-langpack-en '
```

Comment:
This is redundant with what you have in the template file and should be dropped.